docs/user-guide/global-configurations/authorization/permission-groups.md

```diff
@@ -62,25 +67,47 @@ Provide the information in the following fields:
 
 You can add multiple rows for Devtron app permission.
 
-Once you have finished assigning the appropriate permissions for the groups, Click `Save`.
+Once you have finished assigning the appropriate permissions for the groups, Click **Save**.
+
+### Jobs
+
+In `Jobs` option, you can provide access to a group to manage permission for jobs created using Devtron.
+
+Provide the information in the following fields:
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/permission-group/permission-groups-jobs-v2.jpg)
+
+| Dropdown | Description |
+| --- | --- |
+| **Project** | Select a project from the drop-down list to which you want to give permission to the group. You can select only one project at a time.<br>Note: If you want to select more than one project, then click `Add row`.</br> |
+| **Job Name** | Select the specific job name or all jobs from the drop-down list.<br>Note: If you select `All Jobs` option, then the user gets access to all the current jobs including any new job which gets associated with the project later.</br>  |
+| **Workflow**  | Select the specific workflow or all workflows from the drop-down list.<br>Note: If you select `All Workflows` option, then the user gets access to all the current workflows including any new workflow which gets associated with the project later.</br> |
+| **Environment** | Select the specific environment or all environments from the drop-down list.<br>Note: If you select `All environments` option, then the user gets access to all the current environments including any new environment which gets associated with the project later.</br> |
+| **Role**  | Select one of the [roles](#role-based-access-levels) to which you want to give permission to the user:<ul><li>`View only`</li></ul> <ul><li>`Run job`</li></ul><ul><li>`Admin`</li></ul>  |
```

Comment:
Approver roles?

Reply:
This is the **Jobs** section. There's no approver role (as far as I can see, please check the attached snapshot):
<img width="984" alt="image" src="https://github.com/devtron-labs/devtron/assets/141001279/967c668d-ede3-4dae-805b-979c77b75fbd">